### ISSUE TITLE
Make nugen and ucr-icetray use the new propagator service (Trac #276)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#276]
Add the option to use the new propagator service.

Migrated from https://code.icecube.wisc.edu/ticket/276
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T22:29:52", 
    "description": "Add the option to use the new propagator service.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547245792926506", 
    "component": "combo simulation", 
    "summary": "Make nugen and ucr-icetray use the new propagator service", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2011-06-02T18:37:03", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20110607T18:16:15: 
* olivas changed status from "new" to "closed"
* olivas commented:

> Fixed in revisions 76225 and 76229 for neutrino-generator and ucr-icetray respectively.


* olivas changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20141122T18:26:38: 
* olivas commented:

> Milestone IceSim 2.6 Release deleted


* olivas changed milestone from "IceSim 2.6 Release" to ""

**Comment by icecube-trac** [comment]
Trac update at 20190111T22:29:52: nwhitehorn commented:

> In changeset 169507:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169507"
> s/-/_/g for python lib names. fixes #276
> ```
